### ISSUE TITLE
Group content should become a permanent member of the "general" keyboard help section

### ISSUE THREAD

**Issue by zepumph** [phetsims/scenery-phet#630]
From https://github.com/phetsims/joist/issues/668#issuecomment-704242185, since the phet menu uses arrow keys, we should add this everywhere.

Tagging @terracoda and @jessegreenberg, but I'll implement.

**Comment by terracoda** [comment]
In some issue, I think I mentioned this, but I don't remember which one.
Could we re-order the list when you implement, moving "move between items" up?

H2: Basic Actions
UL:
  LI: Move to next item or group with Tab key.
  LI: Move to previous item or group with Shift plus Tab key.
  LI: Move between items in a group with Left and Right arrow keys or Up and Down Arrow keys.
  LI: Press buttons with Space key.
  LI: Toggle checkboxes with Space key. **(I think this one is optional)**
  LI: Exit a dialog with Escape key.

**Comment by zepumph** [comment]
Sure thing, in this issue I will make that ordering, you said the same order over in https://github.com/phetsims/joist/issues/669

Also, I was wondering in the order of things, once we merge the Basic Actions into one, if the order can be like this:
**Basic Actions**
- Move to next item or group with Tab key.
- Move to previous item or group with Shift plus Tab key.
- Move between items in a group with Left and Right arrow keys or Up and Down Arrow keys.
- Press buttons with Space key.
- Toggle checkboxes with Space key.
- Exit a dialog with Escape key.

**Comment by zepumph** [comment]
The `withGroupContent` option was removed above, and always added. There was some changed to max widths where dialogs were increased in width, and went past the sim-specific max width (friction and faradays-law). But that was an easy enough change.

Here are all the sims that had group content added because they are just using the default general help section:
Friction
Faradays Law
GFL:B
Molarity
John Travoltage
RIAW


Here are the string keys that we are no longer using on master, but need to stick around because they are in translated, published sims, and unfortunately rosetta looks at master to discover these values:

* keyboardHelpDialogTabDescriptionString
* keyboardHelpDialogShiftTabDescriptionString
* keyboardHelpDialogMoveToPreviousItemString
* keyboardHelpDialogMoveToNextItemString

I guess these need to stick around forever. . . unused.

**Comment by zepumph** [comment]
Next I will change the order.

**Comment by zepumph** [comment]
@terracoda, everything is done here on master. Please review.

**Comment by terracoda** [comment]
All this sims above look good, except John Travoltage.

The Keyboard Shortcuts dialog of JT looks a bit crowded now. I'll open an issue in the JT repo to address this before next publication.

Screenshot of John Travoltage's latest Keybaord Shortcuts:
![Screen Shot 2020-11-25 at 1 29 20 PM](https://user-images.githubusercontent.com/1902116/100259036-510afc00-2f22-11eb-8ff0-e25dce59cb00.png)

**Comment by terracoda** [comment]
Otherwise all good, so closing.
